bin/export-legacy-workflow

```diff
@@ -12,7 +12,7 @@ require 'ruby-oci8'
 require 'csv'
 
 conn = OCI8.new(ARGV[0], ARGV[1], ARGV[2])
-columns = %w[id druid datastream process status error_msg error_txt datetime attempts
+columns = %w[id druid workflow process status error_msg error_txt datetime attempts
```

Comment:
This is for the legacy database, we haven't renamed it there.

Reply:
Gotcha.